lib/datadog/tracing/diagnostics/environment_logger.rb

```diff
@@ -74,14 +74,14 @@ def version
 
         # @return [String] "ruby"
         def lang
-          Core::Environment::Ext::LANG
+          Diagnostics::Ext::LANG
         end
 
         # Supported Ruby language version.
         # Will be distinct from VM version for non-MRI environments.
         # @return [String]
         def lang_version
-          Core::Environment::Ext::LANG_VERSION
+          Diagnostics::Ext::LANG_VERSION
```

Comment:
I recommend using `Datadog::Core::Environment::Ext::LANG_VERSION` instead of creating yet another `LANG_VERSION` just for this use.

Different ruby runtimes (cruby, jruby, truffleruby) has slightly different semantic for these fields, so it's good to hide all the complexity in one place and let all of Ruby Datadog products get consistent versions values.

I suggest the same for `def lang` above.

Reply:
Good point. Just adjusted both constants, and updated the environment/ext.rb file as well.